oauth-client-util/build.gradle

```diff
@@ -103,7 +95,7 @@ bintray {
         repo = project.group
         name = project.name
         userOrg = 'radar-cns'
```

Comment:
change to radar-base

Reply:
That's not possible, since the bintray org is still called radar-cns.